src/js/components/Data/filter.js

```diff
@@ -60,10 +60,19 @@ export const filter = (data, view, properties) => {
           );
 
         // options case
-        if (Array.isArray(filterValue)) return !filterValue.includes(value);
+        if (Array.isArray(filterValue)) {
+          // value is array case
+          if (Array.isArray(value))
+            // from https://stackoverflow.com/a/39893636
```

Comment:
```suggestion
```
It is fine to use the `.some` without referencing the stack overflow source since it's a generic method.

Reply:
Originally included it because it was the exact code needed to directly solve the issue.